torch/cuda/amp/grad_scaler.py

```diff
@@ -21,6 +21,20 @@ def get(self, device):
         return retval
 
 
+# Defines default_factory for GradScaler's _per_optimizer_states defaultdict,
+# as well as associated "enum" values.  Prefers defining these at top level because
+# - Lambdas can't be pickled, so we don't want to supply a lambda as the factory.
+# - Defining READY, UNSCALED, STEPPED and _refresh_per_optimizer_state within GradScaler
+#   causes a circular reference, which we'd rather avoid.
+READY = 0
```

Comment:
Do we want to make this an enum now that we don't support python 2 anymore? You removed the comment mentioning this.

Reply:
sure, done in https://github.com/pytorch/pytorch/pull/38296/commits/c870c8319d1655276372dda3751e8760558b5e5b.